circle.yml

```diff
@@ -32,16 +34,16 @@ dependencies:
         timeout: 7200
 
     - stack clean
-    - stack --stack-yaml=stack-8.2.2.yaml setup
-    - stack --stack-yaml=stack-8.2.2.yaml build  -j 2 --test --only-dependencies :
+    - stack --stack-yaml=stack-8.0.2.yaml setup
+    - stack --stack-yaml=stack-8.0.2.yaml build  -j 2 --test --only-dependencies :
         timeout: 7200
-    - stack --stack-yaml=stack-8.2.2.yaml build -j 2 :
+    - stack --stack-yaml=stack-8.0.2.yaml build -j 2 :
```

Comment:
This line doesn't make sense to me. Basically we build HIE just to clean it later and rebuild (and make 2x build time)

Reply:
The initial build is to get the deps installed, I think.

And I was getting builds stomping on each other, so perhaps went overboard on this stuff.